projects/ad7768-1fmcz/src/ad77681evb.c

```diff
@@ -135,6 +143,17 @@ int main()
 	Xil_ICacheEnable();
 	Xil_DCacheEnable();
 
+	ret = axi_clkgen_init(&clkgen, &clkgen_init);
+	if (ret != SUCCESS) {
+		printf("error: axi_clkgen_init() failed\n");
+		return FAILURE;
+	}
+
+	ret = axi_clkgen_set_rate(clkgen, 80000000);
```

Comment:
If those values are tied to each other it's probably worthwhile to have a define that we give to both.

Reply:
I will add a define for it.

Reply:
Done.